topics/asciidoc.adoc

```diff
@@ -0,0 +1,123 @@
+[[asciidoc]]
+== AsciiDoc
+http://asciidoc.org/[AsciiDoc] is a text document format that you can use to write documentation, articles, books, web pages, and blogs. AsciiDoc files can be translated to many formats including HTML, PDF, EPUB, and man page.
+
+The Red Hat documentation team uses http://asciidoctor.org/[Asciidoctor] as part of the publishing toolchain.
```

Comment:
Add a ^ before the ] to make links to external sites open in new windows.

Reply:
>We probably want to rename the 'asciidoc' file to 'rendering' and pull the 'publishing' info out of structure to create the 'publishing.adoc' file.  

Someone (Petr?) already made that change.